Fix hasClass on SVG

giorgian opened the pull request:
SVG on some Webkit versions (notably: iOS) don't have attribute classList.

giorgian committed:
Fix: SVG on some Webkit versions (notably: iOS) don't have attribute classList.

Commit:
Test that DOM className helpers work correctly with SVGs.

andreacampi commented:
See https://github.com/andreacampi/treesaver/tree/318-test-case

This test is passing (without your change) on the iOS simulator with iOS 5.0, 5.1, 6.0 and 6.1
Can you try again to replicate this issue?

giorgian committed:
Fix SVG creation in tests.

SVG elements must be created via document.createElementNS, with the correct
namespace ( http://www.w3.org/2000/svg ); if created with document.createElement,
they are not SVG nodes but Unknown Element nodes (i.e. their prototype is
HTMLUnknownElementPrototype instead of SVGSVGElementPrototype).
